lib/potassium/recipes/heroku.rb

```diff
@@ -133,4 +132,18 @@ def run_toolbelt_command(command, app_env_name = nil)
       `heroku #{command} --app #{app_env_name}`
     end
   end
+
+  def valid_heroku_name(name, element, force_suffix = true)
+    suffix = "-#{element}"
+    while name.length > HEROKU_NAMES_MAX_CHARS
+      puts "Heroku names must be shorter than #{HEROKU_NAMES_MAX_CHARS} chars."
+      if force_suffix
+        puts "Potassium uses the heroku-stage gem, because of that '#{suffix}' will be "\
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Rails/Output: Do not write to stdout. Use Rails's logger if you want to log.